Gemfile.lock

```diff
@@ -1,8 +1,8 @@
 PATH
   remote: .
   specs:
-    cocoapods-patch (1.0.3)
-      cocoapods (~> 1.12.0)
+    cocoapods-patch (1.2.0)
+      cocoapods (~> 1.13.0)
```

Comment:
I think you need to run bundle install again to update the lockfile to reflect the gemspecs changes